MDANSE/Src/MDANSE/Framework/Jobs/PositionAutoCorrelationFunction.py

```diff
@@ -90,6 +89,11 @@ def initialize(self):
 
         self.numberOfSteps = self.configuration["atom_selection"]["selection_length"]
 
+        self.labels = [
+            (element, (element,))
+            for element in self.configuration["atom_selection"].get_natoms()
+        ]
```

Comment:
Seeing this a lot, does this just want to become a method where it returns 
```python
list(
    zip(
        self.configuration["atom_selection"].get_natoms(),
        combinations_with_replacement(
          
  self.configuration["atom_selection"].get_natoms(), 
            n=1
        )
    )
)
```
 for this style, n=2 for pairwise?